modules/ROOT/pages/whats-new-in-mule.adoc

```diff
@@ -3,7 +3,8 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Mule 4.3 incorporates several changes from earlier Mule 4.x versions. If you are migrating from a 4.x version to 4.3, ensure that you read each of the relevant topics in this article to make the necessary changes to your applications and also understand what has changed in the Mule runtime engine.
+Mule 4.3 is the latest Mule Runtime which provides a huge performance boost on your applications, while still providing all the benefits of the Mule 4 platform that speeds up the development of applications and resilient runtime for critical mission use cases. 
```

Comment:
```suggestion
The 4.3 version of the Mule runtime engine provides a huge performance boost to your application, while continuing to provide all the benefits of the Mule 4 platform that expedite application development and to offer a resilient runtime for use cases that are vital to the functioning your organization. 
```

Reply:
somehow, somewhere we have to say what the Mule 4 platform is. Maybe we can link to an article on the mulesoft website? Or we have to call each of the products by their official names, which are long.

Reply:
not quite happy with that suggestion yet either. want to clean it up a bit.

Reply:
could say "expedite Mule application development with Anypoint Platform" 

That would avoid having to explain what the Mule 4 platform is